packages/lab/stories/tabstrip-next/tabstrip-next.stories.css

```diff
@@ -0,0 +1,3 @@
+#root {
```

Comment:
Why is this needed? If this impacts story, then the user likely will bump into the same problem and we need document this